packages/discord.js/typings/index.d.ts

```diff
@@ -4705,14 +4710,13 @@ export type MessageChannelComponentCollectorOptions<T extends MessageComponentIn
 export interface MessageEditOptions {
   attachments?: MessageAttachment[];
   content?: string | null;
-  embeds?: (Embed | APIEmbed)[] | null;
```

Comment:
Small request, can you add a test that ensures builders are supported? (both the d.js one and /builders one?)

Reply:
Will try to do that tomorrow, can't finish it today